docs/openapi/components/schemas/credentials/DigitalProductPassportCredential.yml

```
@@ -0,0 +1,224 @@
Lo2uK7xF4osDcyjU+CcwP12y5eZP55HokDdeHpKwRmYfqkUHmrTZI2TyXqysP+wRtEvvM4/iMj8TlWuhJoWy2Zz0a+nNtBP1ArpvaJbpkrp/K2vi+cvlKe3Lvod5foVAoFD591CZBoVAoFAqFQuGB4eb0crt06VInyQFBMCQLQjp4jQsSkQx5gFwI+UAHEAfaQ1ogM0Kok4ecH5Lm0b21MH7tUHzQoauOpLhx695rftjJQ3rQ8YoFviR2iB8pmw23F8ffOEiscv74EqM8fQMfiSe6jptPxKpjV9emDz7ABohXGy1P3Dt/6ZPPkDFigfQpR+xMbphtu9Zv622PGpzbP/qjP2rf+ta3Vj5SbBz41gAS2y/efTNA2XcIPEGQV2qEyMr5RW75m/DJx3CDIH838OTAN7/5zb7h8FnD8Tz33HP9NUCOPePHMSRex2msgHENqUPG0zCRDW2SQoI6N/p0Tp3PQuFhQx/zG2fbsddeb8ujOZEn2KTZmdn+6iHlXK+ztph7riVk5oucXtaurBES5NoD2vlwvcl863GMEv/pg60yqPMpl/QXnST+6GsD/vQrJrrqfNhgpWfttoYH6tHlQ3Kc6VeccjJJn2T6XVwaramPbW7bZx/8E1eFQqHwKKI2CQqFQqFQKBQKDwxet/D6ieOd6EYcIAKQBYgE5AGiNZsDyI+QB4gI8hCNSIph4gNBoYxkUKcXYgPJvn52vr+WCPjjF6KjL/LU+QFyUO9xzMy12fnx9wvoksv1pywG0E6WftQRpKkjk7XzKXYx53jIEg/wn+PMry2di06wjFToStrlwNb5pCuuxJ8NEWXk8BOPHVjp51GB8/knf/InfYPAObBJQGbTJa8YskGAyPare3KEdsYm+Dsav53MGpWzUWBzIK8ZysYBG/aeIPCqodUCx/XVr361k/Z5ikKsyTOec8zDcSSlHv1hip7xlzxj1rl67bXX+kbFZ/lERaHwaWHrhk3thZdf7E8ReIJgefnu+nL31XiQ637ml3pkud5bI8Accy2RZ96lPT4i4991KZvxfGkj4x+G/YB2SAxB+qcL+rcJYB2JjTZyrx1yjQQx0AH9ZjPD/I8811AyvnKMbIfH8MTug21y4tFaowqFQuGzQG0SFAqFQqFQKBQeGOanZtv5a++vEBoIATmSYUhMDH+JGN08SRCSnDwELJJBO0Lj/jYyudf/8JOkTySENmV++UBUxAd59MkhGw3q5MjgxI8UFnvsxLBhbl1/dZEy//Qch7hy/Or6Azqbpte3pcnxUxb0k+gjT8jTR+zIlT15IAbkihiHxxEdvhy72Cc3zbXH1m3pPh4FOCf/5b/8l/bXf/3XnRhHbCGt/LrfpoANAh/4RebnCQJPABhjzpvz5+/g/EnGDXvn3TnPqzeU6UE2CLzCaLXBWPn617/eTp8+3d57771+fpIcm3EDxpxyxlFSxh8My6A9NhnDzqPxyL9XD9kgXA1PVhQKnyRmpqbb1aWb7dTJk21qNN4nR2Mf+pzJhsHdeQZZp4KQ84E5k/UD7p97oB6Zeea6Q58tmOvayVyzXL+Us24m0cm8FZ9Nz8xlfl3XMocl+uQ2W+llQzX98k+PnK02/bimkJOlLyngV4wzW9e3x+YfnTWqUCgUPivUJkGhUCgUCoVC4YHi6sTtduHChRVyAgkQAhZpgLwYEiBIAwSsHKEYkiIEOiJCjkBBNCBoQTl5SA82iHzQD18hJZTFQpc/OZ0QHXI+fM/gzsg1uxA78uiKg8wxdH8jXXIykKvnmLKpkDiUZydH5+XOmDihr03syhLSRps4Q97EVp/xpzyUi5Mf5C8/6l7/dOQR+aWmc/Jf/+t/bf/jf/yPTmhlg8B3Ajw54NsDchsEnijw6iG/tnf+cu5yXtnJszkg+bvYJFCOrr/vb/7mb/bNh9UKY98TBefOnWvnz5+/N+ZGaXI0trJZ4HiNOSljHzL+Mj5jH2iX2BivcomOjxkre2KjUHiYsGnH1vbtb397vEHtOj15dy0a/ZcnCswl1xdQh64zmi9y15G0Zw3IfJICthJdydzyujM21ktzTJmO3HUpaxU/dCR26dt8V87aGl/92jAqi51cUmZjPbEZoB/yXAeUs+bRA2WJbs6D6yo9+uTaXU8f37W/TT3C388pFAqFB4HaJCgUCoVCoVAoPFDMzs21k2ffWSEBkA5IAwSBHPkQkiFtyiEWtfslIntAZCAx6dJTDqkCSAZ26S8bEupeHeRX/tkgQFDkdQnxH4Jk8+S6duXWta7PHyKDvnIIErriC/HBb0gVZX7oZqOCbYhX+kD/9p2xjeNQjx/6+kTwABv9DWPQh7rzQI9crGSgnV6OU5unCXbMPfyvfckGgVcMIfOde08K2BiQhh8qtnFgg8B5k5xnf4OME7m/YzYIbE7xSebc03f+f/3Xf70dPHjwbgSrF47RRoFz8+677/bxt3x3LDtWx27sZGxLxhYZkA9hjCXRoZ+cXc4rnDhxouutxictCoUfFRun59vZKxfbB+9/MKrdGX+fYASvHzIPMv4Dc2C4PknKElgPhmub9lz/ldnnGqWcdTJrqz4zj9lJ0c38VE9ZyrojBtezxK3OTgJxa3ftcw3Ut6Rd//SzpvHLT64Z4ogscjL24uHzzrrptndTbSQWCoXCp4naJCgUCoVCoVAoPFDMTUy1k5fO9nKICIRASHQEAVIgxAdyIcSHJwkQ8XSQD3J2dG0sICniB0J2yPXBV3yz9wFj+tr0gcBAcMQvuToCeHl6HC+CmD8+gG7k9JXZquuPXnQSj77Fy2/8QGIMqcNOGcTBL0Ia+IGcF7qxSRx853hy7M6RYw2pTXb5yuX2uT2HVnw+jLA58Bd/8Rf9FUOIcOfFubAx4MmBbBD4RbuNgzy14vw7x85pxoNzJnf+bLL4xS6fcn935xvB9au/+qvtiSeeuBvB6ocx4mPGi6PjfPvtt//GmNJu7CiTZYwNEf37Qdf45yPzx/mVtOnPufMEx8M8Dgu
```

Change to single-pixel png
```suggestion
        "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z8BQDwAEhQGAhKmMIQAAAABJRU5ErkJggg==,
```

---

^ this is the best suggestion I have ever seen